### PR TITLE
Fixes #18754: Add File content page

### PR DESCRIPTION
**Pull request by ehelms**
Requires -- https://github.com/Katello/bastion/pull/174

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/files/views/files.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  <span data-block="no-search-results-message" translate>
+    Your search returned zero Files.
+  </span>
+
+  <div data-block="table">
+    <table class="table table-striped table-bordered" ng-class="{'table-mask': table.working}">
+      <thead>
+        <tr bst-table-head>
+          <th bst-table-column="name">{{ "Name" | translate }}</th>
```
Comment (ehelms) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/files/views/files.html]: This is the form we use everywhere else for table headers, IIRC due to how the directives clash on transclusion.

**Comment by ehelms** [comment]
@waldenraines addresed all comments

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/files/views/files.html (hunk 1 above)
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/files/views/files.html]: Oh yeah, that's right.  I've been updating to this form everywhere when I was converting from nutupane:

```html
<th bst-table-column="name">
  <span translate>Name</span>
</th>
```

**Comment by waldenraines** [comment]
Should yum filters within the same content view show up in the file filters list?

**Comment by ehelms** [comment]
@waldenraines Updated

**Pull request merged**